Gemfile

```diff
@@ -14,63 +14,33 @@ group :documentation do
   gem 'relish', '~> 0.7.1'
 end
 
-platforms :jruby do
-  gem "jruby-openssl"
-end
+gem 'rake', '~> 12'
```

Comment:
Why not `> 12`, 13 is out?

Reply:
Isn't `~>` a different way to describe the same? When looking at the logs I had an impression it installs 13.0.something.

Reply:
Oh of course, no points, `> 12` is clearer here, `~>` is usually used to install a point release